README.md

```diff
@@ -1,11 +1,11 @@
 We're hiring!
 --------
-Ever thought about joining us?<br/>
```

Comment:
The explicit <br/> is needed to force the line break. If we don't want a break, put it all on one line.